cmd/mutex.go

```diff
@@ -77,9 +112,15 @@ var getSubCmd = &cobra.Command{
 
 // initialization of variables
 func init() {
+	mutexCmd.PersistentFlags().BoolVar(&hasTLS, "tlsauth", false, "Use mTLS auth to connect to etcd, if this is set, --etcdpass and --etcduser are ignored")
+	mutexCmd.PersistentFlags().StringVar(&caCertFile, "cacert", "", "The file containing the CA certificate(to be used with --tlsauth)")
```

Comment:
When it comes to deployment, it would be easier to supply the certs and keys via environment variables (or docker secrets).
https://github.com/spf13/viper#working-with-environment-variables describes how the config file parameters can be overridden by environment variables. This is a good pattern. See cmd/serve.go for how certs and keys were supplied to the server.

Reply:
Yes, that makes sense. Will incorporate the viper pattern.